.gitignore

```diff
@@ -94,3 +94,4 @@ src/scaffolding.config
 # JetBrains Rider
 .idea/
 *.sln.iml
+/src/.attransport/
```

Comment:
that  one is in the wrong place. we should probably also just use .learningtransport as the folder, as that one is already on the gitignore further up